doc/api/cli.md

```diff
@@ -523,8 +523,10 @@ application reference the transpiled code, not the original source position.
 `--enable-source-maps` enables caching of Source Maps and makes a best
 effort to report stack traces relative to the original source file.
 
-Overriding `Error.prepareStackTrace` prevents `--enable-source-maps` from
-modifying the stack trace.
+Overriding `Error.prepareStackTrace` may prevent `--enable-source-maps` from
+modifying the stack trace. Call and return the results of the original
+`Error.prepareStackTrace` in the overriding function to modify the stack trace
+with source maps.
```

Comment:
So in other words, something like this?

```suggestion
modifying the stack trace. Call and return the results of the original
`Error.prepareStackTrace` in the overriding function to modify the stack trace
with source maps.

\`\`\`js
const originalErrorPrepareStackTrace = Error.prepareStackTrace;

Error.prepareStackTrace = (error, stack) => {
  if (Math.random() < 0.5) { // Some condition
  	return originalErrorPrepareStackTrace(error, stack);
  }
  
  stack = stack.map(frame => frame.toUpperCase()); // Some customization
  
  return `${error.toString()}\n${stack.join('\n')}\n`;
}
\`\`\`
```

I have a feeling this might not have been what you meant, in which case an illustrative example would really help. Since the input to `Error.prepareStackTrace` is already more informative than the return value of `originalErrorPrepareStackTrace`, it’s unclear to me what the value is of ever preserving the reference to the original version. (Assuming that I’m understanding the inputs and return values correctly, which is another thing that documentation could help clarify.)

Reply:
Re: https://github.com/nodejs/node/pull/50827#discussion_r1399809362

I think we don't want to expand on the V8 API object and its API stability is not guaranteed by us. But I can see the problem that plain text may not explain well and I've added a simple example.